src/lib/scale/utils.ts

```diff
@@ -77,3 +78,17 @@ export async function getProcurement({
 
   return res.data ?? null;
 }
+
+export function formatColocationStrategy(
+  colocationStratgy: Procurement["colocation_strategy"],
+) {
+  return match(colocationStratgy)
+    .with(
+      { type: "pinned" },
+      ({ cluster_name }) => `\"pinned\" (${cluster_name})`,
+    )
+    .with({ type: "colocate-pinned" }, () => `\"colocate-pinned\"`)
+    .with({ type: "colocate" }, () => `\"colocate\"`)
+    .with({ type: "anywhere" }, () => `\"anywhere\"`)
+    .exhaustive();
```

Comment:
Omega nit but these don't have to be format strings do they? You could just do `'"colocate-pinned"'`